scripts/install-appstudio-e2e-mode.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+#
+# exit immediately when a command fails
+set -e
+# only exit with zero if all commands of the pipeline exit successfully
+set -o pipefail
+# error on unset variables
+set -u
+
+command -v kubectl >/dev/null 2>&1 || { echo "kubectl is not installed. Aborting."; exit 1; }
+command -v oc >/dev/null 2>&1 || { echo "oc cli is not installed. Aborting."; exit 1; }
+
+export MY_GIT_FORK_REMOTE="qe"
+export MY_GITHUB_ORG=${MY_GITHUB_ORG:-"redhat-appstudio-qe"}
+export MY_GITHUB_TOKEN="${GITHUB_TOKEN}"
+export TEST_BRANCH_ID=$(date +%s)
+export ROOT_E2E=$(dirname $(dirname $(readlink -f "$0")));
+export WORKSPACE=${WORKSPACE:-${ROOT_E2E}}
+
+# Download gitops repository to install AppStudio in e2e mode.
+function cloneInfraDeployments() {
+    git clone https://github.com/redhat-appstudio/infra-deployments.git ./tmp/infra-deployments
```

Comment:
Path should be: "../tmp...."
Otherwise I get:
`./install-appstudio-e2e-mode.sh: line 27: cd: /Users/jkopriva/git/flacatus/e2e-tests/tmp/infra-deployments: No such file or directory`

Reply:
No, should be 
`git clone https://github.com/redhat-appstudio/infra-deployments.git "$WORKSPACE"/tmp/infra-deployments`